File: examples/macro_drop/macro_drop_python.cpp
#include <erCv/erCv.hpp>


int main( int hola, char** file_name)
{

  CvRect rect;
  rect.x=20;rect.y=160;rect.width=230;rect.height=75;
  erCerc cerc(20,210,5);
  erEqualP equ(0);
  erSmootP p1(BLUR,7),p2(GAUSSIAN,5);
  erCannyP cann(355,355);
  erAdThrP adp(THRESH_BINARY,AM_MEAN,50,14,255); //** < 
  erMacroDropAnalysis mda("macroDrop");
  mda.defineParameters(rect,cerc,p1,p2,cann,adp,equ);

  mda.doIt("imageExemple_1.bmp");

  return(0);
};

File: examples/macro_drop/macro_drop_python.py
import sys
sys.path.append('../../libs')
from PyerCv import *

rect=CvRect();
rect.x =20;rect.y=160;rect.width=230;rect.height=75;
cerc= erCerc(20,210,5);
p1  = erSmootP(BLUR,7)
p2  = erSmootP(GAUSSIAN,5);
cann= erCannyP(355,355);
adp = erAdThrP(THRESH_BINARY,AM_MEAN,50,14,255);

an=erMacroDropAnalysis("peo","peo")
an.defineParameters(rect,cerc,p1,p2,cann,adp)
an.doIt("imageExemple_1.bmp")
